xbmc/cores/VideoPlayer/Process/ProcessInfo.h

```diff
@@ -110,4 +114,5 @@ class CProcessInfo
   // player states
   CCriticalSection m_stateSection;
   bool m_stateSeeking;
+  std::atomic_int m_levelVQ;
```

Comment:
The default constructor of `std::atomic_int` leaves the object in an uninitialized state. Calling GetLevelVQ() before SetLevelVQ() will result in UB. `m_levelVQ` should be initialized by `CProcessInfo` or the UB should be clearly documented.